feeder/parsers/ENTSOE.py

```diff
@@ -72,11 +72,13 @@ def query_consumption(domain, session):
         'outBiddingZone_Domain': domain,
         'periodStart': now.replace(hours=-24).format('YYYYMMDDHH00'),
         'periodEnd': now.replace(hours=+24).format('YYYYMMDDHH00'),
-        'securityToken': os.environ['ENTSOE_TOKEN']
+        'securityToken': os.environ.get('ENTSOE_TOKEN')
     }
     response = session.get(ENTSOE_ENDPOINT, params=params)
     if response.ok: return response.text
     else:
+        if os.environ.get('ENTSOE_TOKEN') is None:
```

Comment:
Good idea to add an exception. I would however add it before L69, when constructing the `params` dict, as this is where the code will crash if there is no `ENTSOE_TOKEN`.